sklbench/datasets/transformer.py

```diff
@@ -113,8 +115,38 @@ def split_and_transform_data(bench_case, data, data_description):
         "KNeighbors" in get_bench_case_value(bench_case, "algorithm:estimator", "")
         and int(get_bench_case_value(bench_case, "bench:mpi_params:n", 1)) > 1
     )
-    if distributed_split == "rank_based" or knn_split_train:
-        from mpi4py import MPI
+
+    if distributed_split == "sample_shift":
```

Comment:
Please update configs/README.md where "distributed_split" is mentioned and add this to options with some description (https://github.com/IntelPython/scikit-learn_bench/tree/large-scale/configs#data-parameters)